api/src/dev/server/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN apt-get update && \
 
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 
-ENV CLOUD_SDK_VERSION 260.0.0
+ENV CLOUD_SDK_VERSION 321.0.0
```

Comment:
We should update the Circle image as well: https://github.com/all-of-us/workbench/blob/d0f2cb6cf0afef69efae437fe33d0535308c8453/ci/Dockerfile.circle_build#L20

Possibly in this PR? I don't feel strongly. Probably pushing a new version to dockerhub should be out of scope here as this would pull in a handful of other file changes.

Reply:
I'll push a followup for this

Reply:
https://github.com/all-of-us/workbench/pull/4448